iceberg/iceberg-handler/src/test/java/org/apache/iceberg/mr/hive/TestHiveIcebergCTAS.java

```diff
@@ -88,6 +89,35 @@ public void testCTASPartitionedFromHiveTable() throws TException, InterruptedExc
     Assert.assertEquals("name", table.spec().fields().get(1).name());
   }
 
+  @Test
+  public void testCTASTblPropsAndLocationClause() throws Exception {
+    Assume.assumeTrue(HiveIcebergSerDe.CTAS_EXCEPTION_MSG, testTableType == TestTables.TestTableType.HIVE_CATALOG);
+
+    shell.executeStatement("CREATE TABLE source (id bigint, name string) PARTITIONED BY (dept string) STORED AS ORC");
+    shell.executeStatement(testTables.getInsertQuery(
+        HiveIcebergStorageHandlerTestUtils.CUSTOMER_RECORDS, TableIdentifier.of("default", "source"), false));
+
+    String location = temp.newFolder().toURI().toString();
+    shell.executeStatement(String.format(
```

Comment:
Can we add a use-case where the location is provided by the user?

Reply:
This new test contains the user-added location clause: `LOCATION '%s'`
The earlier CTAS tests contain tests without the location clause.